pybossa/api/api_base.py

```diff
@@ -420,6 +420,7 @@ def _update_instance(self, existing, repo, repos, new_upload=None):
             new_data = request.form
         self._forbidden_attributes(new_data)
         self._restricted_attributes(new_data)
+        self._forbidden_attributes_put(new_data)
```

Comment:
There is a `immutable_attributes` property. Is that something we could have used?